templates/watcher/config/00-default.conf

```diff
@@ -59,3 +59,13 @@ datasources = ceilometer
 [cache]
 memcached_servers = {{ .MemcachedServers }}
 {{ end }}
+
+{{ if (index . "PrometheusHost") }}
+[prometheus_client]
+host = {{ .PrometheusHost }}
+port = {{ .PrometheusPort }}
+fqdn_label = fqdn
```

Comment:
why not fqdn then (or that is a followup patch ?) but still as commented elsewhere I don't follow why api needs these (and not decisionengine)